modules/library/xml/src/test/java/org/geotools/xml/GMLParserTest.java

```diff
@@ -355,4 +365,47 @@ public void testProblemFeatures() {
             // fine, they were expected
         }
     }
+
+    @Test
+    public void testSecured() throws Exception {
+
+        // mock URL checker is not configured for tile.openstreetmap.org
+        // should throw exception
+        MockFileURIChecker urlChecker = new MockFileURIChecker();
+        urlChecker.setEnabled(true);
+        URLCheckerFactory.addURLChecker(urlChecker);
+
+        // since the shema loader handles the error
+        // we will assert the security by listening to log
+        Handler exceptionListner =
+                new Handler() {
+
+                    public boolean errorThrown = false;
+
+                    @Override
+                    public void publish(LogRecord record) {
+                        errorThrown |=
+                                record.getMessage().contains("did not pass securityevaluation");
```

Comment:
There should be a space between "securityevaluation", likely the construction of the error message is faulty, fix it along with the test checking it.

Reply:
will fix